model/patch_lifecycle.go

```diff
@@ -625,6 +630,36 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	return patchVersion, nil
 }
 
+// getFullPatchParams will retrieve a merged list of parameters defined on the patch alias (if any)
+// with the parameters that were explicitly user-specified, with the latter taking precedence.
+func getFullPatchParams(p *patch.Patch) ([]patch.Parameter, error) {
+	paramsMap := map[string]string{}
+	if p.Alias != "" && IsPatchAlias(p.Alias) {
+		aliases, err := findAliasesForPatch(p.Project, p.Alias, p)
+		if err != nil {
+			return nil, errors.Wrapf(err, "retrieving alias '%s' for patched project config '%s'", p.Alias, p.Id.Hex())
```

Comment:
for patched project config seems like it shouldn't be part of the alias message, since aliases can also be defined on the project page, and parameters will be added to this eventually